docs/resources/policy_storage.md

```diff
@@ -10,7 +10,52 @@ description: |-
 
 The storage policy resource allows you to configure a Boundary storage policy. Storage policies allow an admin to configure how long session recordings must be stored and when to delete them. Storage policies must be applied to the global scope or an org scope in order to take effect.
 
+## Example Usage
 
+```terraform
+resource "boundary_scope" "global" {
+  global_scope = true
+  scope_id     = "global"
+}
+
+resource "boundary_scope" "org" {
+  name                     = "organization_one"
+  description              = "My first scope!"
+  scope_id                 = boundary_scope.global.id
+  auto_create_admin_role   = true
+  auto_create_default_role = true
+}
+
+resource "boundary_scope" "project" {
+  name                   = "project_one"
+  description            = "My first scope!"
+  scope_id               = boundary_scope.org.id
+  auto_create_admin_role = true
+}
+
+resource "boundary_target" "ssh_session_recording_foo" {
+  name         = "ssh_foo"
+  description  = "SSH target"
+  type         = "ssh"
+  default_port = "22"
+  scope_id     = boundary_scope.project.id
+  host_source_ids = [
+    boundary_host_set.foo.id
+  ]
+  injected_application_credential_source_ids = [
+    boundary_credential_library_vault.foo.id
+  ]
+  enable_session_recording = true
+  storage_bucket_id        = boundary_storage_bucket.aws_example
+}
+
+resource "boundary_policy_storage" "storage_policy_example" {
+  scope_id = boundary_scope.org.id
+  delete_after_days = 1
+  retain_for_days = 0
+  name = storagepolicyexample
```

Comment:
I'm not sure if this needs to be in quotes? otherwise this looks great to me 

```suggestion
  name = "storagepolicyexample"
```

Reply:
Great spot, @stellarsquall ! Thank you